connector_sage/models/res_partner/mapper.py

```diff
@@ -54,6 +54,10 @@ def names(self, record):
         ]
         return {"name": " ".join(parts)}
 
+    @mapping
+    def ref(self, record):
+        return {"ref": str(record["CodigoEmpleado"])}
+
```

Comment:
@KNVx FYI it was necessary to convert it to string, "CodigoEmpleado" is an integer